docs/content/javascript/select2.html

```diff
@@ -1,12 +1,11 @@
 ---
 title: Select2
```

Comment:
We talk about select2 and call it select2 and even go as far as to show example jquery code using select2.... but when someone views the page, that person will not see it in action, just normal `select`s.